libcxx/include/optional

```diff
@@ -210,6 +210,7 @@ namespace std {
 #  include <__iterator/wrap_iter.h>
 #  include <__memory/addressof.h>
 #  include <__memory/construct_at.h>
+#  include <__ranges/enable_borrowed_range.h>
```

Comment:
@smallp-o-p I noticed that you didn't update the synopsis. Can you please update it. I think its preferable if you do it yourself.